### PR TITLE
Add cowsay package.

### PR DESCRIPTION
**Pull request by dlorenc**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [X] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [X] REQUIRED - The version of the package is still receiving security updates
- [X] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

### CODE REVIEW

**Comment by dlorenc** [comment]
```
# cowsay wolfi
 _______
< wolfi >
 -------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
```

**Comment by joshrwolf** [comment]
needs fips

**Comment by dlorenc** [comment]
> needs fips

I think it really needs CUDA drivers so we can GPU-accelerate rendering

File: cowsay.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      # set it to ${{targets.destdir}}/usr, that also ends up in the final perl script.
+      # Use sed to hardcode that part to /usr instead.
+      sed -i 's|%PREFIX%|/usr|g' cowsay
+      mkdir -p ${{targets.destdir}}/usr
+      ./install.sh ${{targets.destdir}}/usr
+
+  - uses: strip
+
+update:
+  enabled: true
```
Comment (ajayk) [inline cowsay.yaml:37]: Do we need the update check as the repository is archived ?

**Review by ajayk — APPROVED** [review]

**Pull request merged**